tests/pytests/test_mdis_drivers.py

```diff
@@ -1,125 +1,473 @@
 import pytest
 import pvl
+import os
+import subprocess
+import numpy as np
+import spiceypy as spice
 
 import ale
-from ale.drivers import messenger_drivers
 from ale.base import data_naif
-from ale.base import label_pds3
-from ale.base import label_isis
 
-from unittest.mock import PropertyMock, patch
-
-# 'Mock' the spice module where it is imported
-from conftest import SimpleSpice, get_mockkernels
-
-simplespice = SimpleSpice()
-
-data_naif.spice = simplespice
-messenger_drivers.spice = simplespice
-label_pds3.spice = simplespice
+# We have to re-assign spiceypy because other tests overwrite it
+ale.base.data_naif.spice = spice
 
 from ale.drivers.messenger_drivers import MessengerMdisPds3NaifSpiceDriver
-from ale.drivers.messenger_drivers import MessengerMdisIsisLabelNaifSpiceDriver
 
-MessengerMdisPds3NaifSpiceDriver.metakernel = get_mockkernels
-MessengerMdisIsisLabelNaifSpiceDriver.metakernel = get_mockkernels
+@pytest.fixture(scope="module", autouse=True)
+def convert_kernels():
+    ale_root = os.path.split(ale.__file__)[0]
+    data_root = os.path.join(ale_root, '../tests/pytests/data/EN1072174528M')
```

Comment:
Is there a better way to get portable data paths?

Reply:
I like this little example: https://github.com/USGS-Astrogeology/plio/blob/master/plio/examples/__init__.py